lib/jobby_job_job/posting_processor.rb

```diff
@@ -27,6 +27,11 @@ def initialize
           url:"https://remoteok.io/remote-dev-jobs.json",
           format:"json",
           parser:"RemoteOk"
+        },
+        {
+          url:"https://stackoverflow.com/jobs/feed",
```

Comment:
Is there a way to set the location to "Remote" so it only returns remote jobs?

Reply:
Yeah like on Indeed, you can put "Remote" into the location field and it should return Remote jobs. I think you can do that on Stack Overflow.

Reply:
ooooooh, i see what you mean..... ok let me take a look at it.

do you think you can maybe marge this first and create a new issue to add the remote filter :D ?
that would be +2 towards the shirt ! haahha

Reply:
haha yeah I can do that for you. Cheers🍻